src/node/performance/worker_service_impl.js

```diff
@@ -36,6 +36,11 @@
 var BenchmarkClient = require('./benchmark_client');
 var BenchmarkServer = require('./benchmark_server');
 
+exports.quitWorker = function quitWorker(call, callback) {
+  callback(null, {});
+  process.exit(0);
```

Comment:
So the intention is that this is a separate worker method that gets invoked after the driver is done with the last one. I guess there's an issue here. In C++ we have a lock in the worker such that a worker process can only be running one of the RPCs (RunClient, RunServer, CoreCount, QuitWorker) at a time. Is there an equivalent in Node.js ?